lib/circles/create.js

```diff
@@ -3,10 +3,10 @@ const fullStateEnum = require("./fullStateEnum");
 module.exports = function(req, res) {
     const circle = {
         name: req.body.name || null,
-        vision: req.body.vision || null,
-        mission: req.body.mission || null,
-        aim: req.body.aim || null,
-        expectationsForMembers: [],
+        vision: req.body.vision || "",
+        mission: req.body.mission || "",
+        aim: req.body.aim || "",
+        expectationsForMembers: req.body.expectationsForMembers || "",
```

Comment:
I am not sure I like having defaults to empty strings. If they're not there, they are not supposed to be there at all. We are using a document-storage scheme anyway. Hmm.